plugins/org.jkiss.dbeaver.ui.editors.base/src/org/jkiss/dbeaver/ui/editors/NodeEditorInput.java

```diff
@@ -75,7 +77,8 @@ public boolean exists() {
 
     @Override
     public ImageDescriptor getImageDescriptor() {
-        return DBeaverIcons.getImageDescriptor(getNavigatorNode().getNodeIconDefault());
+        DBNNode node = getNavigatorNode();
+        return node == null ? null : DBeaverIcons.getImageDescriptor(node.getNodeIconDefault());
```

Comment:
Now we have three places with potential NPE for `getImageDescriptor()` call 🥲
<img width="853" height="381" alt="image" src="https://github.com/user-attachments/assets/37057cae-2316-4df3-8477-842a851186af" />

Reply:
Well, the javadoc clearly says that it's allowed to return `null`. But judging by those calls, neither of them seems to be affected by my changes